resources/sharedconfig.ini

```
@@ -65,4 +65,15 @@ version_minor=64
 
 # The port the chat server is started and listening on
 # Used in chat and world servers
-chat_server_port=2005
\ No newline at end of file
+chat_server_port=2005
+
+sqlite_database_path=resServer/dlu.sqlite
+
+# If using sqlite, set this to 1 and the others to 0
+using_sqlite=1
+
+# If using mysql, set this to 1 and the others to 0
+using_mysql=0
+
+# Skips the account creation check in master. Used for non-interactive setups.
+skip_account_creation=1
```

Shouldn't this be

```suggestion
skip_account_creation=0
```
 here so that users get it and then =1 just in `.env.default`

---

we dont have a .env.default file, should I create one?

---

do you mean .env.example?

---

I added these to .env.example and updated the docker stuff to read these values.  is this what was wanted?